dts/bindings/clock/raspberrypi,pico-syscon.yaml

```diff
@@ -0,0 +1,48 @@
+# Copyright (c) 2022 Andrei-Edward Popa
+# SPDX-License-Identifier: Apache-2.0
+
+description: Raspberry Pi Pico clock controller node
+
+compatible: "raspberrypi,pico-syscon"
```

Comment:
Is syscon the preferred postfix?
`syscon.yaml` exists, but it is not using that definition.
IMO, `clock-controller` (or `clock` related) postfix seems better here.

Reply:
Yeah, I think you are right, I used clocks now